js/content.js

```diff
@@ -19,19 +19,13 @@ import {
 
 import { createLyricView, clearLyricView } from './views/lyricsView';
 
-const getLyrics = () => {
```

Comment:
This function was doing a lot more than getting lyrics

Reply:
Agreed.